Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+pipeline{
+  agent{
+    node { label 'master' }
+  }
+  options{
+    timestamps()
+    ansiColor('xterm')
+  }
+  stages{
+    stage('Checkout Source'){
+      steps{
+        checkout scm
+      }
+    }
+      stage('PR Builder'){
+      steps{
+        sh "build/mvn install"
```

Comment:
@markhamstra @ianlcsd would we need anything other than just a mvn install? or should i be including other flags

Reply:
see other comment